Documentation/csr-metrics.md

```diff
@@ -0,0 +1,8 @@
+# CSR Metrics
+
+| Metric name| Metric type | Labels/tags | Status |
+| ---------- | ----------- | ----------- | ----------- |
+| kube_csr_created| Gauge | `csr`=&lt;csr-name&gt;| STABLE |
```

Comment:
```suggestion
| kube_certificatesigningrequest_created| Gauge | `csr`=&lt;csr-name&gt;| STABLE |
```

I understand that this may be concerning given the length, but I see instances like `kube_persistentvolumeclaim_resource_requests_storage_bytes`